src/user.js

```diff
@@ -172,7 +167,7 @@ export default function createUserModule({ web3, utils }) {
       },
     });
 
-    return await utils.requestAPI({
+    return utils.requestAPI({
```

Comment:
this change affects myxo, right?

Reply:
Nope, not at all. I removed it because is useless declare a function as "async" and "await" the returned promise. An async function will always return a promise so makes no sense to await any response in the end.